tests/helpers/util.py

```diff
@@ -71,6 +73,65 @@ def run_agent(config_text):
             yield [fake_services, get_output, c]
 
 
+def print_dp_or_event(dp_or_event):
```

Comment:
I would put this in another file called `formatting.py` or something like that.  The util module is already pretty overloaded with stuff.